### PR TITLE
fix: forgotten impacted files by PR #3278

### PR DESCRIPTION
**Pull request by MelReyCG**
Just a quick-fix: added generated files changes & impacted exemple by the PR #3278, as I encountered the following compilation error :
```
/GEOS/inputFiles/singlePhaseFlowFractures/fractureFlow_conforming_2d_vtk_input.xml:26: element VTKMesh: Schemas validity error : Element 'VTKMesh', attribute 'surfacicFieldsInGEOSX': The attribute 'surfacicFieldsInGEOSX' is not allowed.
/GEOS/inputFiles/singlePhaseFlowFractures/fractureFlow_conforming_2d_vtk_input.xml fails to validate
XML validation failed. See details in xml_validation_results.log
make[2]: *** [CMakeFiles/geosx_validate_all_xml_files.dir/build.make:71: CMakeFiles/geosx_validate_all_xml_files] Error 1
make[1]: *** [CMakeFiles/Makefile2:2199: CMakeFiles/geosx_validate_all_xml_files.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

### CODE REVIEW

File: inputFiles/singlePhaseFlowFractures/fractureFlow_conforming_2d_vtk_input.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" ?>
 
-<Problem>
+<Problem xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+ xsi:noNamespaceSchemaLocation="../../build-pangea4-gcc12.1-hpcxompi2.17.1-onemkl2023.2.0-release/schema.xsd">
```
Comment (jhuang2601) [inline inputFiles/singlePhaseFlowFractures/fractureFlow_conforming_2d_vtk_input.xml]:
```suggestion
<Problem>
```

**Review by jhuang2601 — APPROVED** [review]
Thanks for fixing it

File: inputFiles/singlePhaseFlowFractures/fractureFlow_conforming_2d_vtk_input.xml (hunk 1 above)
Comment (MelReyCG) [reply inputFiles/singlePhaseFlowFractures/fractureFlow_conforming_2d_vtk_input.xml]: Whoops! I forgot to remove that.

**Review by CusiniM — APPROVED** [review]

**Comment by MelReyCG** [comment]
Closed as other PRs fixed it.

**Pull request closed without merging**